doc/api/http2.md

```diff
@@ -1389,6 +1418,26 @@ per session. See the [Compatibility API](compatiblity-api).
 added: v8.4.0
 -->
 
+#### Event: 'checkContinue'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `request` {http2.Http2ServerRequest}
+* `response` {http2.Http2ServerResponse}
+
+Emitted each time a request with an HTTP `Expect: 100-continue` is received.
+If this event is not listened for, the server will automatically respond
+with a status `100 Continue` as appropriate.
+
+Handling this event involves calling [`response.writeContinue()`][] if the client
+should continue to send the request body, or generating an appropriate HTTP
+response (e.g. 400 Bad Request) if the client should not continue to send the
+request body.
+
+Note that when this event is emitted and handled, the [`'request'`][] event will
+not be emitted.
```

Comment:
We need to specify that this event will be emitted only if there is a `'request'` event handler, or a function is passed through  `createServer` / `createSecureServer`.

Reply:
I prefer the first one.

Reply:
Agreed. The first one

Reply:
Agreed. Wanted to make sure it wasn't weird starting with the 'if'. 👍 Updated and committed.